web/generate-geometries.js

```diff
@@ -580,11 +596,44 @@ zoneFeatures = toListOfFeatures(zoneFeaturesInline);
 // Write unsimplified list of geojson, without state merges
 fs.writeFileSync('build/zonegeometries.json', zoneFeatures.map(JSON.stringify).join('\n'));
 
-// Simplify
+// Adds the zones from topo2 to topo1
+function mergeTopoJsonSingleZone(topo1, topo2) {
```

Comment:
can you move the function definition above the script part (to line ~550)?

Reply:
OK

Reply:
I changed to moreDetails, zonesMoreDetails and topoMoreDetails that seems better for me as well.